components/Loader/index.js

```diff
@@ -17,7 +18,7 @@ function CircularIndeterminate (props) {
 	return (
 		<div>
 			{isLoading &&
-          <CircularProgress className={classes.progress} size={100}/>
+				<CircularProgress className={classes.progress} size={100}/>
```

Comment:
It is unrelated to what you proposed to work. However, I just wonder how we can ensure consistent code indentation. Any idea?

Reply:
Is there any specific folder that we should check with `eslint`? 
Maybe we could add a command into `package.json`.
If anyone wants to check their code, they could just run the `npm run eslint`.

Reply:
I ran the `eslint components` to check the code in the component folder, there are more inconsistent code in the component folder.
Should I fix them in this pull request? 🤔

Reply:
Let's concentrate on coding style consistency in next PR. Please do.